homeassistant/components/group/light.py

```diff
@@ -74,7 +72,10 @@
 )
 
 SUPPORT_GROUP_LIGHT = (
-    SUPPORT_EFFECT | SUPPORT_FLASH | SUPPORT_TRANSITION | SUPPORT_WHITE_VALUE
+    LightEntityFeature.EFFECT
+    | LightEntityFeature.FLASH
+    | LightEntityFeature.TRANSITION
+    | SUPPORT_WHITE_VALUE
```

Comment:
SUPPORT_WHITE_VALUE probably needs to be dropped, but I think that should be done in a separate PR (preliminary or follow-up) by someone who is more knowledgeable than me.

Reply:
It is here for compatibility reasons at this point